src/cli/traceViewer/web/ui/networkResourceDetails.css

```diff
@@ -0,0 +1,112 @@
+/*
+  Copyright (c) Microsoft Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+.network-request {
+  box-shadow: var(--box-shadow);
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  margin-bottom: 10px;
+  background: #fdfcfc;
+  width: 100%;
+  border: 3px solid transparent;
+  flex: none;
+  outline: none;
+}
+
+.network-request.selected,
+.network-request:hover {
+  border-color: var(--inactive-focus-ring);
+}
+
+.network-request.selected:focus {
+  border-color: var(--orange);
+}
+
+.network-request-title {
+  height: 36px;
+  display: flex;
+  align-items: center;
+  flex: 1;
+}
+
+.network-request-title-status {
+  font-weight: bold;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0px 5px;
+  margin-right: 5px;
+}
+
+.status-success {
+  background-color: rgb(124, 187, 62);
+}
+
+.status-failure {
+  background-color: rgb(211, 35, 35);
```

Comment:
Could you please use some colors from common.css? If you need another one, let's introduce it.

Reply:
I'm assuming you meant the variables in `styles.tsx` file? If so, I added two variables there, and used them in the changes I added to the network tab.